elasticsearch/install.sh

```diff
@@ -8,19 +8,27 @@ source ${HOME}/prep-install.${RELEASE_STREAM}
 echo "ES plugins: ${es_plugins[@]}"
 for es_plugin in ${es_plugins[@]}
 do
-  ${ES_HOME}/bin/elasticsearch-plugin install -b $es_plugin
+  if [ -x ${ES_HOME}/bin/elasticsearch-plugin ] ; then
```

Comment:
Is this so we can use the install script between 2.x and 5.x?

Reply:
no - the rpm install names it `elasticsearch-plugin` but the es base image uses `plugin`

Reply:
Why does the base image call it plugin?  If you download the artifact for 5.x its called `elasticsearch-plugin`? Is it being renamed by us?

Reply:
@jcantrill yes it is being renamed by us - what difference does it make?

Reply:
I would suspect that we would prefer to modify the original source as little as possible but f it works then I guess its immaterial